### PR TITLE
[4.0] update drone YAML for drone 1.0

### PR DESCRIPTION
**Pull request by SniperSister**
### Summary of Changes
Adjusted the drone YAML for the new syntax of drone 1.0, thanks to @Hackwar for the help!

### CODE REVIEW

**Review by wilsonge — APPROVED** [review]

File: .drone.yml [hunk 1]
```diff
@@ -1,27 +1,31 @@
 ---
+kind: pipeline
+name: default
+
 clone:
-  git:
-    image: plugins/git
-    depth: 1
+  depth: 42
```
Comment (wilsonge) [inline .drone.yml:6]: does it really need to go that deep?

Comment (SniperSister) [reply .drone.yml:6]: Well, with lower numbers we ran into issues with failed builds in the main repo that need to be repeated. The checkout takes about 10 seconds with that setup, so compared to all the other steps it doesn't make a difference

**Comment by SniperSister** [comment]
@HLeithner please review and merge!

**Comment by HLeithner** [comment]
thx

**Pull request merged**